content/W2015/W15-event-coop.md

```diff
@@ -0,0 +1,19 @@
+Title: Co-op Showcase Panel
+Date: 2015-01-09 15:00
+Category: Events
+Slug: co-op-panel
+Status: draft
```

Comment:
We should publish this ASAP, even if it's incomplete, because we don't have much lead time to the event.